### PR TITLE
SQL/SCRIPTS: Death Scythe NQ/HQ Drain Additional Effect

### PR DESCRIPTION
**Pull request by waytim**
Set at 10%~ activation as per wiki discussion.

### CODE REVIEW

File: scripts/globals/items/death_scythe.lua [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+-- Additional Effect: Drains HP
+-----------------------------------------
+require("scripts/globals/status");
+
+-----------------------------------
+-- onAdditionalEffect Action
+-----------------------------------
+function onAdditionalEffect(player,target,damage)
+    local chance = 10;
+    if (target:getMainLvl() > player:getMainLvl()) then
```
Comment (TeoTwawki) [inline scripts/globals/items/death_scythe.lua]: NO. I am sick of explaining these are NOT level corrected and that even if it were this method of attempting level correction isn't even anywhere near accurate.

Comment (TeoTwawki) [inline scripts/globals/items/death_scythe.lua]: See every add effect pull request ever.

Comment (TeoTwawki) [inline scripts/globals/items/death_scythe.lua]: On melee weapons with added effects:
* on status effect: fixed chance + chance to resist 
(in which case there is no message the effect just never happens)

* on element damage: fixed chance, and fixed damage that is modified ONLY by elemental resistance 
(fire dmg vs water elemental = you see the log say you hit for zero)

Drain effects work as if they were dark element dmg.

Comment (TeoTwawki) [inline scripts/globals/items/death_scythe.lua]: I apologize for the tone-in-text I just gave you..Got annoyed at seeing it yet again.

**Comment by waytim** [comment]
Amended as discussed with Teo.

**Pull request merged**